script/build-newsletter-editions.js

```diff
@@ -96,14 +96,16 @@ async function saveNewsletterEditions(organizationId, letterheadData) {
   for await (let newsletter of letterheadData.items) {
     let headline = shared.cleanContent(newsletter.title);
 
-    if (!newsletter.publicationDate) {
```

Comment:
I'm not sure why the left side shows the main branch with old code, the left side should match [what main already has](https://github.com/news-catalyst/next-tinynewsdemo/blob/main/script/build-newsletter-editions.js#L99).